recipes/tabix/0.2.5/meta.yaml

```diff
@@ -0,0 +1,26 @@
+package:
+  name: tabix
+  version: '0.2.5'
+
+source:
+  url: https://downloads.sourceforge.net/project/samtools/tabix/tabix-0.2.5.tar.bz2
+  sha256: 6ce0da41a7ef82aa34db413602cc7cd11c08d606f36ebba793da41ec402e2f86
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - zlib
```

Comment:
Because you need GCC right? Explicit is better then implicit and adding it will use the conda GCC not the container GCC.